site/src/Consumer/EmailNotificationConsumer.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Consumer;
+
+use App\Util\Notification;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Class EmailNotificationConsumer
```

Comment:
Can you make this summary line more descriptive, e.g. "Send out notifications over email"

Reply:
That comment needs resolving in a updated push.